internal/install/discovery/psutil_discoverer.go

```diff
@@ -67,3 +70,41 @@ func (p *PSUtilDiscoverer) Discover(ctx context.Context) (*types.DiscoveryManife
 
 	return &m, nil
 }
+
+func filterValues(m types.DiscoveryManifest) types.DiscoveryManifest {
+	if !isValidOpenInstallationPlatform(m.Platform) {
```

Comment:
Should the default be nil instead of empty string? I wonder if we'll find some usages for empty strings later on.

Reply:
This would require updating the type to a string pointer.  Aside from the fact that the empty string would currently also be rejected by graphQL, idiomatic Go says to use normal strings unless nil is necessary. Normal strings are easier and safer to use, without the need to check that`*string` has a value before dereferencing.

Reply:
No problem, thanks for the details.